packages/appstore/src/components/cfd-account-manager/cfd-account-manager.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Icon, Button, Text } from '@deriv/components';
+import { formatMoney } from '@deriv/shared';
+
+type TCFDAccountManager = {
+    appname: string;
+    platform: string;
+    amount: string;
+    currency: string;
+    loginid: string;
+    onClickTopUp: () => void;
+    onClickTrade: () => void;
+};
+
+const CFDAccountManager = ({
+    appname,
+    amount,
+    currency,
+    loginid,
+    platform,
+    onClickTopUp,
+    onClickTrade,
+}: TCFDAccountManager) => {
+    return (
+        <div className='cfd-account-manager'>
+            <div className='cfd-account-manager-icon'>
```

Comment:
I think it's better to follow scss class name conventions:

```suggestion
            <div className='cfd-account-manager__icon'>
```